ryan.gemspec

```diff
@@ -18,10 +18,13 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
+  spec.required_ruby_version = '>= 1.9.3'
+
   spec.add_development_dependency "bundler", "~> 1.10"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec", ">= 3.2", "< 4"
 
   spec.add_dependency 'ruby_parser', '>= 3.7.0', '< 4.0.0'
   spec.add_dependency 'ruby2ruby', '~> 2.2'
+  spec.add_dependency 'pry', '~> 0.10.4'
```

Comment:
Lol, absolutely! I am so used to development deps being last in Gemfiles and gemspecs that I just dropped it in as the last one.